deploy/lambda/auto-deployment.js

```diff
@@ -0,0 +1,115 @@
+const https = require('https');
+
+exports.handler = async function (event, context) {
+    if (!process.env.AUTO_DEPLOY_ENABLED) {
+        console.log('Auto Deploying disabled');
+        return;
+    }
+
+    try {
+        const autoDeployUsername = process.env.AUTO_DEPLOY_USERNAME;
+        const autoDeployToken = process.env.AUTO_DEPLOY_TOKEN;
+
+        if (!autoDeployUsername) {
```

Comment:
Do we want to return early if we are missing these values `autoDeployUsername` or `autoDeployToken`

Reply:
Moved above the try catch